controllers/cruisecontroltask_types.go

```diff
@@ -62,71 +52,112 @@ func (t *CruiseControlTask) Apply(instance *kafkav1beta1.KafkaCluster) {
 	if t == nil || instance == nil {
 		return
 	}
-
+	//nolint:exhaustive
```

Comment:
**Recommended**: We **SHOULD** also leave a comment here why it is required to override the linter for future readers.

Example:
```suggestion
	// nolint:exhaustive // Note: Not all CC operations require updates to the KafkaCluster status.
```